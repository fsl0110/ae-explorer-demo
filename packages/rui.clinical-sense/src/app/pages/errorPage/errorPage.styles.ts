import styled from "styled-components";

// basic exaple for styling html tags
export const ErrorPageStyles = styled.h1`
  background: #000;
  color: #fff;
  padding: 1rem;
`;
