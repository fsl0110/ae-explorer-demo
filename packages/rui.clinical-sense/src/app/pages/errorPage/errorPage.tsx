import React from "react";
import {ErrorPageStyles} from "./errorPage.styles";

export const ErrorPage = () => <ErrorPageStyles>OUPS... something went wrong!</ErrorPageStyles>;
