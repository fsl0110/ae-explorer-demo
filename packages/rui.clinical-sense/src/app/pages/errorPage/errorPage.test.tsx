import React from "react";
import {shallow} from "enzyme";
import toJson from "enzyme-to-json";
import {ErrorPage} from "./errorPage";

describe("<ErrorPage>", () => {
  it("Snapshot", () => {
    expect(toJson(shallow(<ErrorPage />))).toMatchSnapshot();
  });
});
