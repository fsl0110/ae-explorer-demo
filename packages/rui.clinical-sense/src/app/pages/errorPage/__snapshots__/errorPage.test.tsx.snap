// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorPage> Snapshot 1`] = `
<styled.h1>
  OUPS... something went wrong!
</styled.h1>
`;
